src/utilities/layer-from-json.ts

```diff
@@ -0,0 +1,22 @@
+import * as layer from '../layer'
+import { get } from 'lodash'
+import { ILayer, ILayerJSON } from '../layer/base-layer';
+
+export default function layerFromJSON(jsonLayer: ILayerJSON, inputLayer1?: ILayer) { 
+  if (!layer.hasOwnProperty(jsonLayer.type)) return null;
+
+  const Layer: Function = get(layer, `${layer}.${jsonLayer.type}`);
+
+  // eslint-disable-next-line
+  const realLayer: ILayer | any = Reflect.construct(Layer, arguments)
```

Comment:
Can we not cast result for construct to `ILayer`?

Reply:
We can but then `feed-forward.ts` breaks due to the possible early returned null from layerFromJSON.